pkg/infrastructure/services/deployment.go

```diff
@@ -66,10 +56,10 @@ func createRequiredDeployment(service v1alpha1.KogitoService, image string, defi
 							},
 							Env:             service.GetSpec().GetEnvs(),
 							Resources:       service.GetSpec().GetResources(),
-							LivenessProbe:   defaultProbe,
-							ReadinessProbe:  defaultProbe,
+							LivenessProbe:   &probes[0],
```

Comment:
Can't we have a structure for liveness/readiness instead of a slice ?

Reply:
It's an internal thing, that's why I haven't created it. Will do to make it more clearer.